app/assets/javascripts/discourse/mixins/load-more.js.es6

```diff
@@ -21,6 +21,7 @@ export default Mixin.create(Scrolling, {
     const eyeline = new Eyeline(this.eyelineSelector + ":last");
     this.set("eyeline", eyeline);
     eyeline.on("sawBottom", () => this.send("loadMore"));
+    eyeline.update(); // Force update once to consider current position
```

Comment:
This should be in another PR/commit as it's not really related to the group refactoring (remember we squash all the commits when we merge a PR)